src/Trafi.BigQuerier/Dispatcher/DispatcherService.cs

```diff
@@ -52,18 +52,19 @@ private struct QueueItem
         public TimeSpan? LastBatchSendTime { get; set; }
         public int QueueSize => _queue.Count;
 
-        public DispatcherService(
-            IBigQueryClient client,
+        public DispatcherService(IBigQueryClient client,
             TableSchema schema,
             string datasetId,
             Func<DateTime, string> tableNameFun,
+            int batchSize,
```

Comment:
Why doesn't it have a default value to not cause breaking changes?

Reply:
There's another constructor do not break ABI